.settings/.gitignore

```diff
@@ -0,0 +1,3 @@
+/.jsdtscope
```

Comment:
Let's not add gitignore files for any specific IDE. You can add them to your global `.gitignore` file